.github/workflows/e2e-dispatch.yaml

```
@@ -165,15 +165,15 @@ jobs:
       group: e2e-byodb-tests-${{ github.workflow }}-${{ github.head_ref || github.run_id }}
       cancel-in-progress: true
 
-  e2e-db-backup-restore-test:
+  e2e-db-backup-restore-tests:
     needs: [ wait-for-images, should-dispatch ]
-    uses: ./.github/workflows/e2e-db-backup-restore-test.yaml
+    uses: e2e-db-backup-restore-tests.yaml
```

actionlint (1.7.12)

[error] reusable workflow call "e2e-db-backup-restore-tests.yaml" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)